packages/github-oauth/github_client.js

```diff
@@ -25,12 +25,17 @@ Github.requestCredential = (options, credentialRequestCompleteCallback) => {
 
   const loginStyle = OAuth._loginStyle('github', config, options);
 
+  let allowSignup = '';
+  if (Accounts._options?.forbidClientAccountCreation)
+    allowSignup = '&allow_signup=false'; // https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps#parameters
```

Comment:
I know it's not necessary, but just for the sake of convention, can you add `{}` to this `if`? 

```suggestion
  if (Accounts._options?.forbidClientAccountCreation) {
    allowSignup = '&allow_signup=false'; // https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps#parameters
  }
```